exporter/exporter.go

```diff
@@ -95,8 +96,71 @@ func EcsClient() (client *ecs.Client) {
 	return client
 }
 
+func NewRedisClient() (client *r_kvstore.Client) {
+	var i alicloudAccessConfig
+	var err error
+	if os.Getenv("ALICLOUD_DEFAULT_REGION") == "" ||
+		os.Getenv("ALICLOUD_ACCESS_KEY") == "" ||
+		os.Getenv("ALICLOUD_SECRET_KEY") == "" {
+		//get rolename
```

Comment:
For the credential management, you can refer here https://github.com/aws/aws-sdk-go/tree/master/aws/credentials and learn from it.